k8s/object.bzl

```diff
@@ -136,12 +136,14 @@ def _common_impl(ctx):
   files = [ctx.executable.resolver]
 
   cluster_arg = ctx.attr.cluster
+  cluster_arg = ctx.expand_make_variables("cluster", cluster_arg, {})
```

Comment:
For the sake of completeness, can you add this for `namespace` and `image_chroot` as well?  (any anything else I might be forgetting).